modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -592,7 +744,7 @@ FeatureFloodCount::flood(const DofObject * dof_object, int current_idx, int live
   _entities_visited[current_idx][entity_id] = true;
 
   // Determine which threshold to use based on whether this is an established region
-  Real threshold = (live_region > -1 ? _step_connecting_threshold : _step_threshold);
+  Real threshold = feature ? _step_connecting_threshold : _step_threshold;
 
   // Get the value of the current variable for the current entity
   Number entity_value;
```

Comment:
No, it's a superset of Real. If libMesh is configured for Imaginaries,, then this type will be Imaginary. I'll fix it.